tools/rna_tools/cmv/cmcv.xml

```diff
@@ -0,0 +1,68 @@
+<tool id="CMCV" name="CMCV" version="@VERSION@.1">
+    <description>
+        CMCV is a tool for visualization of comparisons between RNA family models (CM)
+    </description>
+   <macros>
+        <import>macros.xml</import>
+    </macros>
+    
+    <expand macro="requirements" />
+
+    <expand macro="stdio" />
+    
+    <expand macro="version" />
+    <command><![CDATA[
+    CMCV
+    @COMMON_ARGS@
+    -m $model_filepath
+    -r $cmcompare_file
+    -p `pwd`
+    > 'warnings'   
+    ]]></command>
+    
+    <inputs>
+      <param name="model_filepath" type="data" format="cm" label="Input model"/>
+      <param name="cmcompare_file" type="data" format="cmcompare" label="Input comparison"/>
+      <section name="common" title="Common parameters">
+        <expand macro="common_parameters" />
+      </section>
+    </inputs>
+    
+    <outputs>
+      <collection name="output_pdfs" type="list" label="${tool.name} on ${on_string}: ">
+        <discover_datasets pattern="^[a-zA-Z0-9_-]*.pdf$" format="pdf"/>
```

Comment:
Does it make sense to output pdfs, svgs, pngs ... all at once. Maybe the user can just choose one output?

Reply:
Thanks for reviewing :-D 
The user can actuallty chose one of the output formats. I just did not know who to adjust the output detection according to the selected input.

Reply:
try this: https://github.com/galaxyproject/tools-iuc/pull/1486/files#diff-6635a1ff814dc622d8a72e03fd8382d3R36